src/indexApi.ts

```diff
@@ -374,6 +374,30 @@ function checkForKeys(data: any): void {
     )
     throw new Error('Log includes sensitive data')
   }
+
+  const regex = /(\b[a-z]+\b\s){11,23}\b[a-z]+\b/g
+  const matches = dataString.match(regex)
+  const found = matches != null
+  if (found) {
+    for (const match of matches) {
+      const words = match.split(' ')
+      let allValid = true
+      for (const word of words) {
+        if (!wordlists.english.includes(word)) {
+          allValid = false
+          break
+        }
+      }
+      if (allValid) {
+        slackPoster(
+          `Log attempt rejected due to mnemonic seed detected ${words
+            .slice(0, 3)
+            .join('-')}`
+        ).catch(e => console.log(e.message))
+        throw new Error('Log includes sensitive mnemonic data')
+      }
+    }
+  }
```

Comment:
This seems fair to be de-duped into a util?

Reply:
done w/ new commit